android/app/src/main/AndroidManifest.xml

```diff
@@ -16,22 +16,14 @@
       <activity
         android:name=".MainActivity"
         android:label="@string/app_name"
-        android:launchMode="singleTop"
         android:screenOrientation="portrait"
-        android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+        android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
+        android:launchMode="singleTask"
         android:windowSoftInputMode="adjustResize">
         <intent-filter>
             <action android:name="android.intent.action.MAIN" />
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
       </activity>
-      <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
-      <service android:name="io.invertase.firebase.messaging.RNFirebaseMessagingService">
```

Comment:
I think we've said this somewhere else already. but this means that we can't send notifications, right?

Reply:
Yes, this boils down to RNfirebase having to be upgraded to v6, in which notification management has been extracted to a separate component. I will have to put them back in, for the third time :(